### ISSUE TITLE
Flags for Kapacitor nodes: stateChangesOnly & flapping

### ISSUE THREAD

**Issue by danisabad** [influxdata/chronograf#1245]
It would be nice to have ability to enable/disable following nodes via flags in web-ui:
- .stateChangesOnly()
- .flapping()

**Comment by nhaugo** [comment]
Blocked by #1236

**Comment by nhaugo** [comment]
• Create a list of options from the kapacitor documentation 
• Then point this story

**Comment by goller** [comment]
Backend complete in #2447

**Comment by russorat** [comment]
closing due to inactivity
